api/courses/coursesRouter.js

```diff
@@ -225,19 +225,8 @@ router.get('/', authRequired, function (req, res, next) {
  *       description: 'Course Instance with id {course_id} does not exist'
  */
 
-router.get('/:course_id', authRequired, checkCourseExists, function (
-  req,
-  res,
-  next
-) {
-  const { course_id } = req.course;
-  try {
-    Courses.findByCourseId(course_id).then((course) => {
-      res.status(200).json(course);
-    });
-  } catch (err) {
-    next(err);
-  }
+router.get('/:course_id', authRequired, checkCourseExists, (req, res) => {
+  res.status(200).json(req.course);
```

Comment:
Nice job refactoring the code down! This is so much cleaner than having the model function inside.